src/renderers/common/Textures.js

```diff
@@ -179,6 +179,7 @@ class Textures extends DataMap {
 				}
 
 				this.delete( renderTarget );
+				this.backend.delete( renderTarget );
```

Comment:
This delete is not necessarily required since the bakcend data are maintained in a weak map but it's still more correct to remove the data as soon as we can.